### DISCUSSION TITLE
Save a route and traveled path logger.

### DISCUSSION THREAD

**Discussion by RootInit** [organicmaps/organicmaps#5175] [Ideas]
I use this app for walking/running frequently and I

1. Often want to save a path to re-take again in the future
2. Want to see where all I actually went

Are there any future feature plans for either of these features by any chance?

**Comment by biodranik** [comment]
There are already related plans/issues on Github.

**Reply by biodranik** [reply]
See https://github.com/organicmaps/organicmaps/issues/1785
